packages/main/src/plugin/learning-center/learning-center.ts

```diff
@@ -0,0 +1,26 @@
+/**********************************************************************
+ * Copyright (C) 2023 Red Hat, Inc.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ *
+ * SPDX-License-Identifier: Apache-2.0
+ ***********************************************************************/
+
+import type { Guide } from './learning-center-api.js';
+import { default as guidesJson } from './guides.json';
+
+export class LearningCenter {
+  static get guides() {
+    return guidesJson.guides as Guide[];
+  }
```

Comment:
not sure why we need a class if you use only static stuff inside

Reply:
Just to avoid undefined check, but if I use `with {type: "json"}` it would not be needed.